e2e-tests/cypress/support/page-objects/management/Reports.ts

```diff
@@ -0,0 +1,118 @@
+import { KibanaNavigation } from '../KibanaNavigation';
+import semver from "semver/preload";
+import { getKibanaVersion } from "../../helpers";
+
+export class Reports {
+  static navigateTo() {
+    cy.log('navigateTo');
+
+    // Check if we are already on the Stack Management page
+    cy.url().then(url => {
+      if (!url.includes('/app/management')) {
+        // If not, open Kibana navigation and click on Stack Management
+        KibanaNavigation.openKibanaNavigation();
+        cy.contains('Stack Management').click();
+      }
+
+      // Now, click on Reporting (whether we were already on Stack Management or not)
+      cy.contains('Reporting').click();
+    });
+  }
+
+  static waitForAllReportsToBeDone(reportTitle: string = 'Untitled discover search') {
+    cy.log('waitForAllReportsToBeDone');
+
+    cy.get('tr[data-test-subj="reportJobRow"]')
+      .each($reportItem => {
+        cy.wrap($reportItem)
+          .find('[data-test-subj="reportingListItemObjectTitle"]')
+          .should('contain.text', reportTitle);
+
+        cy.wrap($reportItem)

```

Comment:
Let's move it to the another method of Reports page-object

Reply:
There is no need for this anymore, there are no differences in the code between versions.